.changelogs/issue_2281.yml

```diff
@@ -0,0 +1,6 @@
+significance: minor
```

Comment:
```suggestion
significance: major
```

Reply:
per our slack conversation let's include this in v7 and call it a major change

Reply:
<3